primitives/application-crypto/src/lib.rs

```diff
@@ -200,9 +198,8 @@ macro_rules! app_crypto_public_full_crypto {
 				$crate::codec::Encode,
 				$crate::codec::Decode,
 				$crate::RuntimeDebug,
-				$crate::max_encoded_len::MaxEncodedLen,
+				$crate::codec::MaxEncodedLen,
 			)]
-			#[max_encoded_len_crate($crate::max_encoded_len)]
```

Comment:
Not sure whether or not it will work to just remove this. If some pallets have trouble building, then the replacement should probably look something like

```rust
#[codec(crate = $crate::codec)] 
```

... That may not work; the key-value assignments in the `#[codec(crate = ...)]` parser currently expect the value to be a string literal, not a path literal. That shouldn't be too hard to fix, though.